image-layout.md

```diff
@@ -64,12 +61,28 @@ For example, an image may have a tag for different versions or builds of the sof
 In the wild you often see "tags" like "v1.0.0-vendor.0", "2.0.0-debug", etc.
 Those tags will often be represented in an image-layout repository with matching refs names like "v1.0.0-vendor.0", "2.0.0-debug", etc.
 
-This illustrates the expected contents of a given ref, the manifest list it points to and the blobs the manifest references.
+### Example Ref
 
 ```
 $ cat ./refs/v1.0
 {"size": 4096, "digest": "sha256:e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7fc331f", "mediatype": "application/vnd.oci.image.manifest.list.v1+json"}
 ```
+
+This illustrates the expected contents of a given ref, the manifest list it points to and the blobs the manifest references.
```

Comment:
Looking at this line more closely after it was [shifted][1], what is it actually saying?  “contents of a given ref” sounds like “a descriptor”, so ok.  “the manifest list it points to” might be the descriptor again.  But how is this illustrating “thte blobs the manifest references”?  Maybe we should drop this line entirely.

[1]: https://github.com/opencontainers/image-spec/pull/423#discussion_r85557101